exporters/otlp/otlp_metric_test.go

```diff
@@ -695,13 +700,78 @@ func TestResourceInstLibMetricGroupingExport(t *testing.T) {
 	)
 }
 
+func TestStatelessExportKind(t *testing.T) {
+	type kase struct {
+		name  string
+		ikind otel.InstrumentKind
+		tempo metricpb.AggregationTemporality
+		mono  bool
```

Comment:
Might be useful to use full words for these so new users are not confused.

Reply:
Done.